twitter/View Models/DetailsModel.h

```diff
@@ -0,0 +1,26 @@
+//
+//  DetailsModel.h
+//  twitter
+//
+//  Created by Angelina Zhang on 6/23/22.
+//  Copyright © 2022 Emerson Malca. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "Tweet.h"
+#import "DetailsView.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface DetailsModel : NSObject
```

Comment:
This is good but the name threw me a bit. Usually when I see 'model' in a name I expect it to be modeling some data in some way. Example, a ViewModel typically just looks like the view but has pure data fields:

```
class SomeView {
  @IBOutlet var nameLabel: UILabel!
}

struct SomeViewModel {
   let name: String
}
```

What's here is more of a DetailsViewDecorator? It takes a model and a view and uses them together.